pkg/kp/pcstore/consul_store.go

```diff
@@ -500,7 +500,14 @@ func (s *consulStore) getInitialClusters(syncer ConcreteSyncer) (WatchedPodClust
 			s.logger.WithField("pc_id", id).Errorln("Error retrieving pod cluster from consul")
 			return prevResults, err
 		}
-		prevResults.Clusters = append(prevResults.Clusters, &existing)
+
```

Comment:
I think this whole function could actually just be simplified - I don't see any reason to individually retrieve each pod cluster anymore to test for existence, given that the value we put in `prevResults` will always be the same. We should just fill `prevResults` with `&fields.PodCluster{ID: id}` in this loop and call it a day.

Is there any reason not to?